packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -18,74 +18,111 @@ import AngleLeftIcon from '@patternfly/react-icons/dist/esm/icons/angle-left-ico
 
 ## Examples
 
-### Basic
+### Basic menus
+
+A menu may contain multiple variations of `<MenuItem>` components. The following example shows a few different states of menu items, where they may:
+
+- Use the `itemId` property to link to callbacks. In this example, the `onSelect` property logs information to the console when a menu item is selected. In practice, specific actions can be linked to `onSelect` callbacks.
+- Use the `onClick` and `to` properties to direct users to other resources or webpages after selecting a menu item.
+- Use the `isDisabled` property to disable a menu item. 
 
 ```ts file="MenuBasic.tsx"
 ```
 
 ### With icons
 
+Use the `icon` property to add a familiar icon before a `<MenuItem>` to accelerate text label recognition.
+
 ```ts file="MenuWithIcons.tsx"
 ```
 
-### With checkbox
-
-```ts file="./MenuWithCheckbox.tsx" isBeta
-```
+### With actions
 
-### Filtering with text input
+Add a `<MenuItemAction>` to a `<MenuItem>` to trigger an action when that item is selected. 
```

Comment:
Went with something like this, but structured a little differently than you suggested, so lmk what you think. I also switched this title to "with action icon" since the basic example shows an action item without an icon. Does that sound accurate?

Reply:
It's looking good! The example title could go either way. Right now we have other components with examples that just use "With actions", so it may depend which is clearer in terms of the purpose of the example.

Otherwise I think the only nit I have with the updated description is maybe instead of "To execute a callback...", we could use your original verbiage of "To trigger an action..."? What do you think?